src/card/card.js

```diff
@@ -83,7 +83,9 @@ function Card(props: CardsPropsT) {
             {title}
           </Title>
         )}
-        <Body {...getOverrideProps(BodyOverride)}>{children}</Body>
+        <Body $as="div" {...getOverrideProps(BodyOverride)}>
```

Comment:
why as div?

Reply:
According to SEO docs, only the text content goes in a `p` but the card examples wrap a `Body` in a `Body`. I guess it can be done as `p` inside a `p`.